blackbox/docs/admin/system-information.rst

```diff
@@ -1199,8 +1199,8 @@ statements.
 | ``failed_count``             | Total number of queries that failed to complete    | ``LONG``         |  
 |                              | successfully.                                      |                  | 
 +------------------------------+----------------------------------------------------+------------------+
-| ``sum_of_durations``         | Sum of durations of all executed queries per       | ``LONG``         |
-|                              | statement type.                                    |                  |
+| ``sum_of_durations_millis``  | Sum of durations, expressed in milliseconds, of    | ``LONG``         |
```

Comment:
It seems a bit odd to me that all other columns (`stdev`, `mean`, etc..) don't have a `_millis` suffix. Given that we can't change the others without a breaking change I'd be in favor of dropping `_millis` here as well for consistency.

Reply:
I thought about that as well, but then it also feels odd to not follow a good practice only to follow an _existing_ one. @joemoe what's your take on this?

Reply:
The problem I see is that if one of them has a `_millis`, as a user you might imply that the others must have another time-unit. So instead of improving things, it could make them worse.

Reply:
you got my point wrong. sorry to be not explicit enough. 

we have `ms` for a lot of values. we should be explicit about this also for sum of durations. that was my point.

That's what i'm referring to:

> The mean query latency in ms
> The maximum query latency in ms
> The minimum query latency in ms

Reply:
@joemoe @mfussenegger please see new commit. thanks